api/internal/utils/utils.go

```diff
@@ -37,10 +38,13 @@ func init() {
 		}
 		salt = uint16(i)
 	}
-
+	ips, err := getLocalIPs()
+	if err != nil {
+		panic(err)
+	}
 	_sf = sonyflake.NewSonyflake(sonyflake.Settings{
 		MachineID: func() (u uint16, e error) {
-			return sumIP(GetOutboundIP()) + salt, nil
+			return sumIP(randomIP(ips)) + salt, nil
```

Comment:
how about sum all local ips?

Reply:
I think one is enough, it is for machineID, one or all IPs are the same :)

Reply:
may it meet 127.0.0.1 ？

Reply:
No, `127.0.0.1` is filtered out

Reply:
👌

Reply:
Here is a risk that different instance may be get the same IP in multiple network environment  (such as virtual network ), calculating all IP would be better when we can not confirm which is physical network interface.

Reply:
OK, you convinced me.
Done.